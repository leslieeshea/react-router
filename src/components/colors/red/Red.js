import React from 'react';

export default function Red() {
  const style = {
    backgroundColor: 'red',
    width: '50vh',
    height: '50vh'
  };

  return <div style={style}></div>;
}
