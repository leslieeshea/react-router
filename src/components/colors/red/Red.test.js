import React from 'react';
import { shallow } from 'enzyme';
import Red from './Red';

describe('Red component', () => {
  it('renders a red splotch', () => {
    const wrapper = shallow(<Red />);
    expect(wrapper).toMatchSnapshot();
  });
});
