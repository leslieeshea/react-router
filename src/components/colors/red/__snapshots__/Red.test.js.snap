// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Red component renders a red splotch 1`] = `
<div
  style={
    Object {
      "backgroundColor": "red",
      "height": "50vh",
      "width": "50vh",
    }
  }
/>
`;
